shared/gh/js/controller/gh.admin-batch-edit.js

```diff
@@ -295,6 +329,15 @@ define(['gh.api.event', 'gh.api.groups', 'gh.api.series', 'gh.api.util', 'gh.adm
                 }
             }
         });
+
+        // Apply jEditable to the event notes
+        $('.gh-jeditable-events-select').editable(editableEventTypeSubmitted, {
+            'cssclass' : 'gh-jeditable-form',
+            'placeholder': '',
+            'select': true,
+            'tooltip': 'Click to edit event notes',
+            'type': 'event-type-select'
```

Comment:
Can you add a callback here that focuses the original element when the form has been submitted? Currently focus is lost.